### PR TITLE
add membership discount new booking created by portal web

### PR DESCRIPTION
**Pull request by VictorZ94**

### CODE REVIEW

File: src/components/AddNewBooking.js [hunk 1]
```diff
@@ -143,7 +148,8 @@ const AddNewBooking = ({ baseElement, classes, coordinators, customers, handleMo
           company: customer ? customer.company : newCompany,
           distributorId,
           utm_source: 'opsPortal',
-          hasInternationalAttendees
+          hasInternationalAttendees,
+          membershipDiscount: membershipDiscount?.active ? membershipDiscount.discount : 0
```
Comment (jarojasres) [inline src/components/AddNewBooking.js:152]: parseFloat(membershipDiscount.discount)

File: src/components/AddNewBooking.js [hunk 2]
```diff
@@ -105,6 +105,11 @@ const AddNewBooking = ({ baseElement, classes, coordinators, customers, handleMo
         return;
       }
 
+      let membershipDiscount = undefined;
+      if (customer) {
+        membershipDiscount = await getUserMembershipDataByEmail(customer.email);
```
Comment (jarojasres) [inline src/components/AddNewBooking.js:110]: Utilizar newEmail

File: src/components/AddNewBooking.js (hunk 1 above)
Comment (VictorZ94) [reply src/components/AddNewBooking.js:150]: it was not necessary.

File: src/components/AddNewBooking.js (hunk 2 above)
Comment (VictorZ94) [reply src/components/AddNewBooking.js]: Done!!

**Review by jarojasres — APPROVED** [review]

**Review by carloskiron — APPROVED** [review]

**Pull request merged**